site/docs/foundations/duration.mdx

```diff
@@ -5,11 +5,13 @@ sidebar:
   priority: 10
 ---
 
-Duration represents the time between a user performing an action and the system giving feedback. Consistency in timing provides comfort and can increase trust in a system.
+Duration represents the time between a user performing an action and the system giving feedback. It’s essential to indicate to users that something is happening when they interact with an element. Consistency in timing provides user comfort and can increase trust in a system. The duration should match the significance of the animated element or information.
 
-## Duration scale
+For detailed information on how to apply duration in design and the use cases, please refer to [this blog post](). (Internal users only)
```

Comment:
Will tick once the link available

Reply:
NIT blog post should be titled, rather than saying 'this blog post'.